Userland/Libraries/LibWeb/CSS/PercentageOr.h

```diff
@@ -97,10 +97,10 @@ class PercentageOr {
 
     CSSPixels to_px(Layout::Node const& layout_node, CSSPixels reference_value) const
     {
-        return resolved(layout_node, Length::make_px(reference_value)).to_px(layout_node);
+        return resolved(layout_node, reference_value).to_px(layout_node);
     }
 
-    T resolved(Layout::Node const& layout_node, T const& reference_value) const
+    Angle resolved(Layout::Node const& layout_node, Angle reference_value) const
```

Comment:
This is confusing me. A lot more types can be PercentageOr than just Angle. If this is meant to prevent calling it for LengthPercentage, can we instead add a requires clause to block that use?

Reply:
Sure, but there are no places where it is used for anything other than Angle. Do you think we should have kept it as a template anyway?